lib/between_meals/changes/cookbook.rb

```diff
@@ -44,19 +38,66 @@ def self.explode_path(path, cookbook_dirs)
         nil
       end
 
+      def self.symlinked_dir?(link_path)
+        File.symlink?(link_path) &&
+          File.directory?(File.absolute_path(link_path))
+      end
+
+      def self.map_symlinks(files)
+        # For each symlink get the source path, if any files have changed under
+        # the source path, fake them as coming from the symlink path. This
```

Comment:
You still need to update the comment here. As mentioned, this block creates a hash of *every* symlink in the repo.